Automattic/woocommerce-services
# Upgrade react to 16.13

Upgrade WCS's React to 16.8+ so that we can start using hooks.

Closes https://github.com/Automattic/woocommerce-services/issues/1944

### Requires/Installation
- node 10
- npm ci //delete node_modules and install based on lock/shrinkwrap

### What have been done
-  Ran `npm install --save react@^16.13.0 react-dom@^16.13.0` 
- Test label process to make sure WCS is not impacted.

### Test
- Run `npm run test` and tests should pass
- Run `npm run test:e2e` and tests should pass

### Manual test printing label
1. Add a product to checkout. Purchase the item.
2. Go to order page and open up the order
3. Create a shipping label. Should be successful
4. Refund label, should be successful.

### Manual test printing label
1. Go to WCS settings page https://localhost/wp-admin/admin.php?page=wc-settings&tab=shipping&section=woocommerce-services-settings
2. Able to update/add credit card. Change label size. Add/update/delete packages
